src/common/image.c

```diff
@@ -1716,6 +1703,20 @@ gboolean dt_images_already_imported(const gchar *filename)
   return result;
 }
 
+uint32_t dt_image_get_id(uint32_t film_id, const gchar *filename)
+{
+  uint32_t id = 0;
```

Comment:
initialized to -1 as I'm pretty sure that for filmroll an image id of -1 is considered not found as the first imported image into the db is 0.